modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -71,6 +76,22 @@
 
 public class MuleContainer {
 
+  private static Logger LOGGER = LoggerFactory.getLogger(MuleContainer.class.getName());
+
+  private static final String PROPERTY_SECURITY_MODEL = SYSTEM_PROPERTY_PREFIX + "security.model";
```

Comment:
Remove this. Use `MULE_SECURITY_SYSTEM_PROPERTY` already defined in `MuleSystemProperties`.